context/app/static/js/shared-styles/tables/ClickableRow/style.js

```diff
@@ -13,7 +13,7 @@ const StyledRow = styled(TableRow)`
       : css`
           cursor: pointer;
           &:hover {
-            filter: ${props.theme.palette.white.hover};
+            background-color: ${props.theme.palette.white.hover};
```

Comment:
As I see this has been repeated in a few places, It seems intentional. Does this have the same visual effect?

Reply:
I swapped these over from filters to colors with the goal of having only colors in the palette, but given that this didn't work as expected upon closer review, I'll revert it and add a note to the theme to indicate that the `hover` palette members should be `filter` values instead of `color` values.